README.md

```diff
@@ -1,12 +1,123 @@
 # SmartKickers
 
-Definition of Done
-
-- Working code
-- Created code need to be merged
-- Tests implemented
-- Tests passed
-- Code review by devs
-- Sanity check with mentors
-- Acceptance criteria fulfilled 
-- Work presented and accepted
+## Overview
+
+SmartKickers is a project to make your kickers more immersive. It uses Ximea camera to track the ball. 
+
+The following repository contains: 
+* Golang server for processing the data.
+* React client to display game information.
+
+You will also need SmartKickersAI provided by the developers per request. It contains:
```

Comment:
You can link to the SmartKickersAI repo using markdown syntax [SmartKickersAI](https://github.com/HackYourCareer/SmartKickersAI).